clients/admin-ui/src/features/privacy-requests/privacy-requests.slice.ts

```diff
@@ -53,99 +56,6 @@ export function mapFiltersToSearchParams({
   };
 }
 
-// Subject requests API
```

Comment:
Just refactored the order of the constant declarations in this file for ES Linting purposes.